src/operator/operator_tune-inl.h

```diff
@@ -616,7 +616,7 @@ class UnaryOpTune : public OperatorTune<DType> {
    */
   template<typename OP>
   static duration_t GetBlankWorkloadEx() {
-    std::unique_ptr<DType> tmp(new DType[Super::WORKLOAD_COUNT]);
+    std::unique_ptr<DType[]> tmp(new DType[Super::WORKLOAD_COUNT]);
```

Comment:
It won't.  
It also will almost certainly never actually cause a runtime problem, because I've never come into contact with a problem mixing delete and delete [] when dealing with scalars.
But just for the sake of textbook syntax, I approved this PR.

Reply:
Might be better to change the title to something which is true.

Reply:
This is reported as definitely lost by valgrind memcheck. Please refer to [c++ delete reference](http://en.cppreference.com/w/cpp/memory/new/operator_delete)

Reply:
@cjolivier01 I think it would be more productive and professional not to be argumentative when somebody points out incorrect use of dynamic memory in C++. While I agree with what you say and is true in general, it's clearly undefined behaviour as specified by the standard and can cause memory leaks, corruption and other problems depending on the implementation. https://stackoverflow.com/questions/15073970/why-does-calling-delete-instead-of-delete-on-an-array-of-class-objects-cause-h

Reply:
I am not sure what you're getting at, @larroy. While valgrind may report a mismatch, it is very unlikely that memory is actually being leaked, which is my point.  Every implementation that I know of evetually just call the same C malloc and free() call for delete and deete[].
For example: https://gcc.gnu.org/viewcvs/gcc/trunk/libstdc%2B%2B-v3/libsupc%2B%2B/del_opv.cc?view=markup
So it's fine to change it as a matter of consistency, but saying that it is leaking memory is probably not true.

Reply:
I  agree with you, and I know that this is the common case, unless you override the operator yourself or some library does it. I actually found a bug in the windows C runtime due to a similar issue. I think still it would be better to have it correct in the code base. My question is why argue about incorrect code. Please accept my apologies, I didn't intend to be hostile in any way.